templates/Web/Pages/Vue.MasterDetail/src/components/MasterDetailPage.vue

```diff
@@ -0,0 +1,58 @@
+﻿<template>
+  <div class="col">
+    <div class="row heading">
+      <div class="col">
+        <h3 class="ml-3 mb-4">{{textSampleData.title}}</h3>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12 mt-3">
+        <nav aria-label="breadcrumb">
+          <ol class="breadcrumb bg-white">
+            <li class="breadcrumb-item">
+              <a class="breadCrumbLink" href="#">Master Detail</a>
```

Comment:
I looked at the templates for React and Angular, and it seems like they have it as ReactMasterDetail and AngularMasterDetail respectively. I don't know if it works differently for Vue, but I think it is good to be consistent.

Reply:
The reason I didn't do that for this specific spot is because our components are being shared by multiple views. I think people would prefer simpler templates over having the name right here. Something i could do instead is pass a prop in with the value VueMasterDetail if you feel this is really needed.

Reply:
When you say
> our components are being shared by multiple views

what do you mean? 

I am not too familiar with Vue yet, so correct me if I'm wrong. From my understanding, it seems like VueMasterDetail.vue is the only one that's using the MasterDetailPage component.

Reply:
Do something like this in react vs angular vs vue: Create 5 master details.
Notice how in vue we share the components folder with the views, while in angular/react we don't